addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -1446,7 +1446,7 @@ export function makeActionManager(env, router = _router) {
             }
             throw new InvalidButtonParamsError("Missing type for doActionButton request");
         }
-        if (action.embedded_action_ids && !this.currentController?.config.currentEmbeddedActionId) {
+        if (params.context?.current_embedded_action_id === undefined) {
```

Comment:
This condition is true in two cases :
1) params.context is undefined or does not exist
2) params.context.current_embedded_action_id is explicitly undefined

Is this intended to execute the block if 1) is true ?

Reply:
No, it is not intended. Thanks for pointing it out :)